docs/content/installation/install_hostdefiner_github.md

```diff
@@ -0,0 +1,15 @@
+# Installing the host definer with GitHub
+
+The host definer for IBM® block storage CSI driver can be installed directly with GitHub.
+
+Use the following steps to install the HostDefiner custom resource, with [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+1. Download the custom resource manifest from [GitHub](https://github.com/IBM/ibm-block-csi-operator).
+
+        curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/v1.10.0/config/samples/csi_v1_hostdefiner_cr.yaml > csi_v1_hostdefiner_cr.yaml
+
+2. Install the `csi_v1_hostdefiner_cr.yaml`.
+
+        kubectl apply -f csi_v1_hostdefiner_cr.yaml
+
+After the host definer is installed, it searches for nodes that have not yet been defined on storage, and defines them.
```

Comment:
Also add in the end of the sentence "according to the host definer attributes configured by the user"

Reply:
done

Reply:
I don't think they are called "attributes". @matancarmeli7 ?

Reply:
i think parameters is better - @matancarmeli7  please confirm

Reply:
params can be ok

Reply:
LGTM. will let @roysahar resolve this when they see fit